apidoc/Titanium/UI/iOS/Stepper.yml

```diff
@@ -5,35 +5,40 @@ summary: |
 description: |
     <table id="IOS Stepper">
       <tr>
-        <td><img src="images/stepper/stepper.png" height="50" /></td>
-        <td><img src="images/stepper/stepper_customImages" height="51" /></td>
+        <td><img src="images/stepper/Stepper.png" height="50" /></td>
+        <td><img src="images/stepper/Stepper_customImages.png" height="51" /></td>
       </tr>
       <tr><th colspan="2">iOS</th></tr>
     </table>
 
     A Stepper displays its selected state while it is being pressed.
     You can specify background images for each state on iOS.
-    Use the <Titanium.UI.createstepper> method or a **`<stepper>`** Alloy element to create a stepper 
-    (see [Examples](#!/api/Titanium.UI.iOS.stepper-examples)) .
+    Use the <Titanium.UI.createStepper> method or a **`<stepper>`** Alloy element to create a stepper
+    (see [Examples](#!/api/Titanium.UI.iOS.Stepper-examples)) .
 
     * When assigning a custom image to the `backgroundImage` property, the image must be
     partially or wholly transparent for the background color or background gradient to be visible.
     * Similarly, for an assigned background gradient to show through, the `backgroundColor` may need to be
     set to 'transparent'.
 
-    *Note that the stepper cannot be re-sized.
+    * Note that the stepper cannot be re-sized.
 
 extends: Titanium.UI.View
 since: "5.4.0"
 excludes:
-    events: [dblclick, doubletap, focus, keypressed, longclick, longpress, pinch, postlayout,
-             singletap, swipe, touchcancel, touchend, touchmove, touchstart, twofingertap]
```

Comment:
Yeah ,  all the ones i didn't take out work. dblcick and all the touchEvents. Added an new method to check events, same one as button class. As its just 2 buttons inside the view for the stepper.